streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -1063,8 +1069,10 @@ void to(final TopicNameExtractor<K, V> topicExtractor,
      * A {@link Transformer} (provided by the given {@link TransformerSupplier}) is applied to each input record and
      * returns zero or more output records.
      * Thus, an input record {@code <K,V>} can be transformed into output records {@code <K':V'>, <K'':V''>, ...}.
-     * This is a stateful record-by-record operation (cf. {@link #flatMap(KeyValueMapper) flatMap()} for stateless
-     * record transformation).
+     * This is a stateful record-by-record operation (cf. {@link #flatMap(KeyValueMapper) flatMap()}. Note that it is
```

Comment:
Overall LGTM. But can we format it differently? We should start a new line for each sentence. If we update the docs later, it make the diff simpler to read.

Reply:
Ack